proposals/sar_backscatter.json

```diff
@@ -0,0 +1,78 @@
+{
+    "id": "sar_backscatter",
+    "summary": "Computes backscatter from georeferenced SAR input",
+    "description": "Computes backscatter from georeferenced SAR input.\n\nNote that backscatter computation may require instrument specific metadata that is tightly coupled to the original SAR products. As a result, this process may only work in combination with loading data from specific collections, not with general data cubes.",
+    "categories": [
+        "cubes",
+        "sar"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "description": "Data cube containing raw backscatter input.",
+            "name": "data",
+            "schema": {
+                "subtype": "raster-cube",
+                "type": "object"
+            }
+        },
+        {
+            "description": "The backscatter coefficient to compute, either `sigma0` or `gamma0` (the default).",
+            "name": "backscatter_coefficient",
+            "optional": true,
+            "default": "gamma0",
+            "schema": {
+                "type": "string",
+                "enum": [
+                    "sigma0",
+                    "gamma0"
+                ]
+            }
+        },
+        {
+            "description": "By default, orthorectification is disabled. Set to `true` to enable orthorectification.\n\nThe non-orthorectified products use a simple earth model as provided in the products themselves. This may be sufficient for very flat target areas and is faster to process.",
+            "name": "orthorectify",
+            "optional": true,
+            "default": false,
+            "schema": {
+                "type": "boolean"
+            }
+        },
+        {
+            "description": "The (collection) name of the digital elevation model to use, leave empty to allow the back-end to make a suitable choice. Not all back-ends may offer all digital elevation models. Elevation models are listed as collections, or can be found in back-end specific documentation.",
+            "name": "elevation_model",
+            "optional": true,
+            "default": null,
+            "schema": {
+                "type": [
+                    "string",
+                    "null"
+                ]
+            }
+        }
```

Comment:
Maybe also add a generic `options`  parameter (like in `save_result`) to allow adding various backscatter tuning parameters that are too implementation dependent to be standardized.

Reply:
I'm not sure about this. In principle the idea of allowing additional parameters is good. The difference with options in save_result is that the potential values are defined in /file_formats and clients (e.g. Web Editor) can help with filling the fields. Here, there's no equivalent so users might struggle to give actual values. A back-end could add a schema, but from experience that's likely not happening in most cases.

Reply:
After discussion with @przell, we thought we may just add it later if required and leave the options out for now.

Reply:
fair enough.

FYI: in the POC implementation of VITO backend we currently use this `options` argument for example to pass an implementation detail `tile_size`, which allows us to benchmark and finetune performance.

Reply:
Good to know. We were about to ask whether anyone needs it for SRR1 / has a use case now (then we'd include it now, too).  So if you need it now, we'll add it.

Reply:
> if you need it ...

We need it in the sense that we currently support it for development/benchmarking reasons. It's not clear if we will need it in the end for normal end users and use cases.

As a backend we can easily add support for this of course. But I also added this `options` parameter to the `sar_backscatter` helper (flagged experimental) in the python client: https://github.com/Open-EO/openeo-python-client/blob/daa440f57ec50ef37f09abd68ab409fa18a3fb5d/openeo/rest/datacube.py#L1494-L1497

Reply:
Okay, I meant needed in a (user-oriented) use case. So for internal reasons indeed you are fine to just add it to the process anyway.

Reply:
and I forgot, we currently also have a `to_dB` option in there too, comparable to the `scaling` argument which is still under discussion

Reply:
Scaling has been removed in favor of just computing dB yourself using apply, multiply and log.